src/GraphQL.EntityFramework/Where/ArgumentProcessor_List.cs

```diff
@@ -18,14 +18,14 @@ static IEnumerable<TItem> ApplyToAll<TItem>(this IEnumerable<TItem> items, Func<
         {
             if (ArgumentReader.TryReadIds(getArguments, out var values))
             {
-                var predicate = FuncBuilder<TItem>.BuildPredicate("Id", Comparison.In, values);
-                items = items.Where(predicate);
+                var predicate = ExpressionBuilder<TItem>.BuildPredicate("Id", Comparison.In, values);
+                items = items.Where(predicate.Compile());
```

Comment:
does this run for every execution? if so isnt compile an expensive operation?

Reply:
So i haven't looked at the project in a while, but my line of thinking was to reduce the code complexity.

I don't remember what it was that caused the divergence for the need to for a FuncBuilder and ExpressionBuilder. I believe the Expression is just the object representation of a function delegate and will have to be compiled anyways to be called wherever it is used.

It would be cheaper to solely use function delegates, rather than expressions.